speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/model/RecognizeWithWebsocketsOptions.java

```diff
@@ -234,9 +232,7 @@ public static class Builder {
     private Boolean splitTranscriptAtPhraseEnd;
     private Float speechDetectorSensitivity;
     private Float backgroundAudioSuppression;
-    private Boolean lowLatency;
     private Float characterInsertionBias;
-    private Boolean interimResults;
```

Comment:
Wouldn't that be considered a breaking change?

Reply:
You are right. For the java-sdk this is a breaking change. I hate to have a major release for such a small change but it would be necessary